frontend/app/tests/e2e/role-management/read.spec.ts

```diff
@@ -9,9 +9,9 @@ test.describe("Role management - READ", () => {
     await test.step("check counts", async () => {
       await expect(page.getByRole("link", { name: "Accounts 9" })).toBeVisible();
       await expect(page.getByRole("link", { name: "Groups 2" })).toBeVisible();
-      await expect(page.getByRole("link", { name: "Roles 2" })).toBeVisible();
+      await expect(page.getByRole("link", { name: "Roles 3" })).toBeVisible();
       await expect(page.getByRole("link", { name: "Global Permissions 8" })).toBeVisible();
-      await expect(page.getByRole("link", { name: "Object Permissions 2" })).toBeVisible();
+      await expect(page.getByRole("link", { name: "Object Permissions 3" })).toBeVisible();
```

Comment:
@pa-lem, can we have something else here such as checking if the identified numbers are equal or above a certain number instead of having an exact match that we need to change each time?

Reply:
we could but it means that it may be a bit less accurate

is it supposed to change often?

if so, in the meantime we can do a "higher or equal to 1" to not bother to much, but having an exact match like right now is a bit more accurate

Reply:
I don't know that this part will change a lot, but there are other things that can. We can perhaps have it as an option for the future if more these type of changes become more frequent.